canonicalwebteam/blog/django/views.py

```diff
@@ -86,6 +90,68 @@ def group(request, slug, template_path):
     return render(request, template_path, context)
 
 
+def archives(request, template_path="blog/archives.html"):
+    try:
+        page_param = request.GET.get("page", default="1")
```

Comment:
Can we just remove the `_param` suffix and reassign if needed?

Reply:
sure :+1: